drivers/video/gc2145.c

```diff
@@ -1057,7 +1136,10 @@ static int gc2145_set_fmt(const struct device *dev, struct video_format *fmt)
 
 	/* Check if camera is capable of handling given format */
 	for (int i = 0; i < ARRAY_SIZE(fmts) - 1; i++) {
-		if (fmts[i].width_min == fmt->width && fmts[i].height_min == fmt->height &&
+		if ((fmts[i].width_min <= fmt->width) &&
+			(fmts[i].width_max >= fmt->width) &&
+			(fmts[i].height_min <= fmt->height) &&
+			(fmts[i].height_max >= fmt->height) &&
 		    fmts[i].pixelformat == fmt->pixelformat) {
```

Comment:
Indentation looks strange.
```suggestion
		if ((fmts[i].width_min <= fmt->width) &&
		    (fmts[i].width_max >= fmt->width) &&
		    (fmts[i].height_min <= fmt->height) &&
		    (fmts[i].height_max >= fmt->height) &&
		    (fmts[i].pixelformat == fmt->pixelformat)) {